newapkbuild.in

```diff
@@ -392,6 +420,7 @@ usage() {
 		  -m  Create meson package (Assume meson.build is there)
 		  -p  Create perl package (Assume Makefile.PL is there)
 		  -y  Create python package (Assume setup.py is there)
+		  -r  Crate rust package (Assume Cargo.toml is there)
```

Comment:
I think we should have a `--something=[autotools|cmake|meson|perl|python|rust]` option rather than take a character for each.

Reply:
Sure, would make sense since you can only choose one at a time anyway.